.github/settings.yml

```diff
@@ -12,13 +12,21 @@ branches:
               strict: true
               contexts:
                   - "Static Analysis (ubuntu-latest, 7.4)"
+                  - "Static Analysis (ubuntu-latest, 8.0)"
                   - "Static Analysis (windows-latest, 7.4)"
+                  - "Static Analysis (windows-latest, 8.0)"
```

Comment:
I assume these "required to merge" settings will only apply for future PRs after this is merged, right?

Reply:
yes :)